main/src/main/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidator.java

```diff
@@ -52,6 +50,14 @@ public class AgencyConsistencyValidator extends FileValidator {
   public void validate(NoticeContainer noticeContainer) {
     final int agencyCount = agencyTable.entityCount();
     if (agencyCount < 2) {
+      GtfsAgency agency = agencyTable.getEntities().get(0);
```

Comment:
Technically, there could be zero agencies.  I'd be defensive and iterate over the entries instead.

Reply:
Was this actually resolved?

Reply:
This was my mistake. Since I was added as a contributor I tried to clean up my remotes locally, pushed this to the core repo instead of the original PR branch!  Above, I ignore 0 agencies, warn on 1, and error on > 1